client/gutenberg/extensions/map-block/info-window/index.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Wordpress dependencies
+ */
+
+import {
+	Component,
+	createPortal
+} from '@wordpress/element';
+
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+
+export class InfoWindow extends Component {
+
+	constructor() {
+
+		super( ...arguments );
+		this.closeClick = this.closeClick.bind( this );
+
+	}
+
+	componentDidMount() {
+
+		const { google } = this.props;
+		this.el = document.createElement( 'DIV' );
+		this.infowindow = new google.maps.InfoWindow( {
+      		content: this.el
+    	} );
```

Comment:
There seem to be indentation issues in this file.

Reply:
Couldn't see this one either?

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/27836/commits/1929d54755a21d7ff44905f66f55fb800ba2723d